app/controllers/maker_controller.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'base_controller'
+
+module Routes
+
+  class MakerController < BaseController
+
+    get '/signup' do
+      erb :'maker/signup'
+    end
+
+    post '/signup-details' do
+      @maker = Maker.create(name: params[:name], email: params[:email], username: params[:username], password_hash: params[:password], password_hash_confirmation: params[:password_confirmation])
```

Comment:
Metrics/LineLength: Line is too long. [194/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)